packages/labs/context/src/lib/create-context.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * A Context object defines an optional initial value for a Context, as well as a name identifier for debugging purposes.
+ */
+export type Context<T> = {
+  name: string;
+  initialValue?: T;
+};
+
+/**
+ * An unknown context type
+ */
+export type UnknownContext = Context<unknown>;
```

Comment:
Do we need this? `Context<unknown>` I understand when reading it, `UnknownContext` I have to look at the definition to know what it is, and it's only two less characters.

Reply:
Agreed, reverted to `Context<unknown>` everywhere.